### ISSUE TITLE
1.2.9 --> 1.2.10 upgrade fails

### ISSUE THREAD

**Issue by dsclassen** [Cacti/cacti#3323]
I received these errors today after doing a git pull from 1.2.9 to 1.2.10

```
Error:	200 OK	Reason:
Notice: Undefined index: name in /var/www/html/cacti/lib/installer.php on line 1079
Notice: Undefined index: name in /var/www/html/cacti/lib/installer.php on line 1080
Notice: Undefined index: name in /var/www/html/cacti/lib/installer.php on line 1081
Notice: Undefined index: name in /var/www/html/cacti/lib/installer.php on line 1088
{"Mode":3,"Step":"2","Errors":[],"Eula":"1","Prev":{"Text":"Previous","Step":1,"Enabled":true,"Visible":true},"Next":{"Text":"Next","Step":3,"Enabled":true,"Visible":true},"Test":{"Text":"Test Connection","Step":-4,"Enabled":true,"Visible":false},"Theme":"modern","StepData":{"Sections":{"location":2,"php_modules":2,"php_optional":2,"mysql_timezone":2,"mysql_performance":0,"php_recommends_web":2,"php_recommends_cli":2}},"Html":"

```

**Comment by netniV** [comment]
This is due to a problem with the Cacti Stats package and is related to #3313. This will help in diagnosing a problem we found and how to prevent it causing installation problems.

**Comment by cigamit** [comment]
Duplicate of https://github.com/Cacti/cacti/issues/3330
Its has been resolved by the merging of the fetch branch into the 1.2.x branch and will be included in the 1.2.11 release.

**Comment by mckaygerhard** [comment]
where's the real fix commit due after applied 501009a there's no files or fixeds .. neither commit 8e68706 not fix anything!

**Comment by netniV** [comment]
Actually, it was fixed.  I would suggest researching properly.  The commit you referenced was merely the changelog commit.

**Comment by TheWitness** [comment]
It's here https://github.com/Cacti/cacti/commit/75020f8ba1debc65aa192e5ef61e48c5f3949c45#diff-f1cec582e16b8cf9eb998174425aee9c

**Comment by tomglx** [comment]
The uploaded gz file in commit 75020f8 is broken. It cannot be unzipped by gunzip. Please provide a fixed one for release 1.2.10.

**Comment by TheWitness** [comment]
Just use the one from the 1.2.x branch.
